src/modules/pages/NewOAuthServerPage/NewOAuthServerItem.js

```diff
@@ -20,7 +20,9 @@ class NewOAuthServerItem extends PureComponent {
     }
 
     submit = values => {
-        this.props.saveOAuthServer(this.props.location.pathname, values);
+        const transformedValues = transformFormValues(values, true);
+
+        this.props.saveOAuthServer(this.props.location.pathname, transformedValues);
```

Comment:
function composition would be possible here too, if parameter order does not fit we could reorder it

Reply:
In this case, it would be a bit harder then it looks like at first. I'll investigate and make separate **PR** related to this case.

Reply:
**Issue**: https://github.com/hellofresh/janus-dashboard/issues/110